src/ci/docker/i686-gnu/Dockerfile

```diff
@@ -11,7 +13,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   cmake \
   sudo \
   gdb \
-  xz-utils
+  xz-utils \
+  libssl-dev:i386 \
+  pkg-config
```

Comment:
Oh I think this can actually be elided, we can remove the `./x.py dist` at the end because we don't actually deploy this builder any more (it's missing `DEPLOY=1`)